schemas/org.mate.peripherals-mouse.gschema.xml

```diff
@@ -15,6 +20,11 @@
       <summary>Motion Threshold</summary>
       <description>Distance in pixels the pointer must move before accelerated mouse motion is activated.  A value of -1 is the system default.</description>
     </key>
+    <key name="accel-profile" enum="org.mate.peripherals-mouse.AccelProfile">
+      <default>'default'</default>
+      <summary>Acceleration profile</summary>
+      <description>Acceleration profile for connected mice. The acceleration profile can be either 'adaptive', which adapts the acceleration depending on the mouse movement, or 'flat', which accelerates by a device-specific constant factor derived from the configured pointer speed. If a mouse doesn't support the configured profile, 'default' will be used.</description>
```

Comment:
We can mention the default at the beginning, like GNOME does:

`
<description>Acceleration profile used for connected mice. The acceleration profile can be set to either default ('default') which uses the default acceleration profile for each device, flat ('flat'), which accelerates by a device specific constant factor derived from the configured pointer speed, or adaptive ('adaptive') which adapts the acceleration depending on the mouse movement. If a mouse doesn't support the configured profile, 'default' will be used.</description>`